pkg/report/testdata/starnix/report/0

```diff
@@ -0,0 +1,210 @@
+[00115.277229][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #42   0x000040f33e900984 in fuchsia::main_singlethreaded::<fuchsia::init_logging_for_component_with_executor<starnix_kernel::main::component_entry_point::{closure#0}, starnix_kernel::main::component_entry_point>::{closure#0}, starnix_kernel::main::component_entry_point::{closure#0}, core::result::Result<(), anyhow::Error>> ../../src/lib/fuchsia/src/lib.rs:148 <>+0x32b984
+[00115.277258][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #43   0x000040f33e8e1ae5 in starnix_kernel::main ../../src/starnix/kernel/main.rs:110 <>+0x30cae5
+[00115.277283][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #44   0x000040f33e9002fe in <fn() -> fuchsia_zircon::port::Port as core::ops::function::FnOnce<()>>::call_once /b/s/w/ir/x/w/fuchsia-third_party-rust/library/core/src/ops/function.rs:250 <>+0x32b2fe
+[00115.277313][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #45   0x000040f33e8f55a1 in std::sys_common::backtrace::__rust_begin_short_backtrace::<fn() -> std::process::ExitCode, std::process::ExitCode> /b/s/w/ir/x/w/fuchsia-third_party-rust/library/std/src/sys_common/backtrace.rs:155 <>+0x3205a1
+[00115.277342][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #46   0x000040f33e8d1584 in std::rt::lang_start::<std::process::ExitCode>::{closure#0} /b/s/w/ir/x/w/fuchsia-third_party-rust/library/std/src/rt.rs:166 <>+0x2fc584
+[00115.277367][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #47   0x000040f5319a9298 in std::rt::lang_start_internal::h1f4f8eaba3c227c3 library/core/src/ops/function.rs:284 <libstd-f4212538686b00a8.so>+0x87298
+[00115.277387][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #48   0x000040f33e8d0ff7 in std::rt::lang_start::<std::process::ExitCode> /b/s/w/ir/x/w/fuchsia-third_party-rust/library/std/src/rt.rs:165 <>+0x2fbff7
+[00115.277406][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #49   0x000040f33e8e34c1 in main <>+0x30e4c1
+[00115.277426][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #50   0x00004142ab5cc3f1 in start_main ../../zircon/third_party/ulib/musl/src/env/__libc_start_main.c:64 <libc.so>+0x6b3f1
+[00115.277446][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #51   0x00004142ab5cc63b in __libc_start_main ../../zircon/third_party/ulib/musl/src/env/__libc_start_main.c:263 <libc.so>+0x6b63b
+[00115.277466][1102][1191][core/starnix_runner/kernels:empty_container.cm_fQdnY5b] WARN:    #52   0x000040f33f773831 in _start ../../zircon/system/ulib/c/Scrt1.cc:7 <>+0x119e831
+[00116.143991][1102][1185][core/sshd-host/shell:sshd-0] WARN: debug1: server_input_global_request: rtype keepalive@openssh.com want_reply 1
```

Comment:
Ah, I see! I think I missed that because they are so close (21 vs 20 occurrences) and the unrelated lines above were explicitly synthetic. Thanks for clarifying.